### PR TITLE
Preprocessing and importing

### PR DESCRIPTION
**Pull request by jshoung**
Ignore the first two commits, unrelated to the change

### CODE REVIEW

File: data_collection.py [hunk 1]
```diff
@@ -61,20 +66,23 @@ def dump_csv(data, filename, fmt):
...
 
-        self.inter_packet_times = list(load_csv('inter_packet_times.txt', dtype=float))
-        self.move_start_indices = list(load_csv('move_start_indices.txt', dtype=int))
+        self.inter_packet_times = list(
+            load_csv('inter_packet_times.txt', dtype=float))
+        self.move_start_indices = list(
+            load_csv('move_start_indices.txt', dtype=int))
         self.num_data_points = len(self.sensors_data[0].acc)
 
-        if os.path.exists(os.path.join(self.experiment_dir, 'labels.txt')):
```
Comment (varung97) [inline data_collection.py]: Why was this change made?

File: preprocess.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+    The noise filters are applied in left to right order.
+    """
+    processed_sensors_data = []
+
+    for i in range(NUM_SENSORS):
+        acc = segment.sensors_data[i].acc
+        gyro = segment.sensors_data[i].acc
+
+        for noise_filter in noise_filters:
```
Comment (varung97) [inline preprocess.py]: I take it that you're not too fond of the `compose` pattern?

IMHO, `compose` makes the code both more readable and more concise and in addition also helps when writing test code (see below)

**Review by varung97 — COMMENTED** [review]
I've made a few changes to the code - refactored some things and changed the graphs being plotted when testing the body/grav code. Lemme know if you disagree with any of the changes!

File: data_collection.py (hunk 1 above)
Comment (jshoung) [reply data_collection.py]: My bad I changed it while messing around with the imports and didn't put it back

**Pull request merged**